frontend/src/components/VSkipToContentContainer.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <Component :is="as" ref="containerNode">
+  <Component :is="as" id="main-content" ref="containerNode" tabindex="-1">
```

Comment:
I don't understand why `tabindex` was set to -1 was set here. I could be wrong but I thought the default is to set it to 0.

Reply:
When the user clicks on the "Skip to main content" link, instead of focusing on the first tabbable element, we are now focusing on the element itself. It can be a `main` or a `div`, which is normally not focusable, or at least not focusable in older browsers. 

I just realized that you weren't asking why there is a tabindex at all but why it's set to `-1`. It's because the element does not need to be focusable when you simply press <kbd>Tab</kbd>, it only needs to be focusable by Javascript.

[This article](https://fae.disability.illinois.edu/rulesets/BYPASS_1/) suggests to use the `-1` tabindex. It also shows that sometimes it makes sense to focus the `h1` and not the main element. We might want to do that in the future, if the current implementation does not work well enough.